lua_modules/liquidcrystal/i2c4bit.lua

```diff
@@ -0,0 +1,55 @@
+local I2C4bit = {}
+
+-- I2C GPIO expander bitmap
```

Comment:
On second thought, could these be shoved into `_bus_args` instead of being fixed like this?

Reply:
The reason behind this it that most of the [LCD I2C modules](https://aliexpress.com/wholesale?SearchText=i2c+lcd+arduino) have this pinout. Of course, it could be shoved into `_bus_args`, but wouldn't it make initialization function excessively large?